app/src/main/java/org/simple/clinic/storage/migrations/Migration_6_7.kt

```diff
@@ -51,6 +55,10 @@ class Migration_6_7 @Inject constructor() : Migration(6, 7) {
       FROM `LoggedInUser_v6`
       """)
       database.execSQL("DROP TABLE `LoggedInUser_v6`")
+
+      database.execSQL("PRAGMA legacy_alter_table = OFF")
+      database.execSQL("PRAGMA foreign_key_check")
```

Comment:
we would only to check the `PRAGMA foreign_key_check` when the foreign key constraints were originally enabled, right? Since we have disabled above, do we need this here?

Reply:
We do need it here, it verifies that the foreign key setup is correct and then we can enable them again. This is as per the "Making Other Kinds Of Table Schema Changes" section in the sqlite `ALTER TABLE` document linked earlier.